app/views/users/index.html.erb

```diff
@@ -36,10 +37,12 @@
                   <%= t(".user_list.roles.#{user.role}") %>
                 </td>
                 <td>
-                  <% if user.active? %>
-                    <%= t(".user_list.statuses.active") %>
-                  <% else %>
+                  <% if !user.active? %>
```

Comment:
I think we need a `User#deactivated?`

Reply:
Yeah, that would be nicer to read! Might do this in a separate PR to stop this one getting too bulky :)